packages/dataviews/src/types.ts

```diff
@@ -32,58 +60,67 @@ export interface Field {
 	 * Callback used to retrieve the value of the field from the item.
 	 * Defaults to `item[ field.id ]`.
 	 */
-	getValue?: ( { item }: { item: Item } ) => any;
+	getValue?: ( args: { item: T } ) => string | undefined;
```

Comment:
I don't think a field is necessary a string or undefined.

Reply:
There is a location in the codebase the return of `field.getValue` is used in a context that requires a string, I'll find it and link it here.

Reply:
Apparently after going through everything, I must have made a mistake, it's all builds with a return of `any`.